lib/rouge/lexers/hocon.rb

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*- #
+
```

Comment:
Can you add '# frozen_string_literal: true'

Reply:
Fixed.